egs2/open_li52/asr1/run.sh

```diff
@@ -48,27 +48,28 @@ full_set="${high_resource_test_set} ${mid_resource_test_set} ${low_resource_test
 test_set=${full_set}
```

Comment:
Some changes in the file are your local setup.
You should revert them.

Reply:
Oh! Missed that. Reflected.